okcupyd/json_search.py

```diff
@@ -26,7 +27,11 @@ def SearchFetchable(session=None, **kwargs):
 
     :param session: A logged in session.
     :type session: :class:`~okcupyd.session.Session`
+    :param order_by: Expected values: 'match', 'online', 'special_blend'
```

Comment:
I get that this is not exactly a filter, but I don't see a good reason that it can't be expressed as one. Lets try to express as much of this in the filters interface as possible.